### PR TITLE
fix: chat responsiveness &  Auth box Scrolling

### PR DESCRIPTION
**Pull request by Uthra-karunakaran**
## Related Issue
The support chat was not responsive , covering the entire screen of the device

Closes: #[issue number]

## Description of Changes
- Made changes to make the support chat responsive by adding a close button to it separately.
- this button is only for sm screens only
- doesn't cover the entier screen
- adjusted the chat to be scrollable in the small screen
- adjusted the margins to fit the sm sizes .

## Checklist:

- [ ] My code adheres to the established style guidelines of this project.
- [ ] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
|orgchat | chat1 |

### CODE REVIEW

**Comment by anmode** [comment]
you can pass prettier test with this,
`npx prettier --write .`

**Comment by anmode** [comment]
https://github.com/user-attachments/assets/67fd8a51-b369-4e61-9931-c580956c5b2e


Just a small fix: this crosss bar should be uniform.. why we replaced it with purple cross with alignment in down

**Comment by Uthra-karunakaran** [comment]
> Screen.Recording.2024-07-23.at.9.32.24.PM.mp4 
> Just a small fix: this crosss bar should be uniform.. why we replaced it with purple cross with alignment in down

i did it coz u see that when user logs in we display their name in the header with that , the cross in the header becomes very small and i thought it was not a good ux , it'll be hard to use that's why i added the cross btn separately below to enhance the ux
chat1

**Comment by anmode** [comment]
> > Screen.Recording.2024-07-23.at.9.32.24.PM.mp4
> > Just a small fix: this crosss bar should be uniform.. why we replaced it with purple cross with alignment in down
> 
> i did it coz u see that when user logs in we display their name in the header with that , the cross in the header becomes very small and i thought it was not a good ux , it'll be hard to use that's why i added the cross btn separately below to enhance the ux chat1

Agreed! Fine, then could you do this will all screen sizes just to have continuity of design?

**Comment by anmode** [comment]
Screenshot 2024-07-26 at 1 31 46 PM
I think the chat box.. our message should be on right..? ok let's plan to re-create the chatbox, integrating AI to it..  let's plan to take this change a step further.

**Comment by anmode** [comment]
I'll create a requirement Doc on the support chat. Like how we can enhance it functionality-wise and UI-wise. by today EOD

**Comment by Uthra-karunakaran** [comment]
> I'll create a requirement Doc on the support chat. Like how we can enhance it functionality-wise and UI-wise. by today EOD

sure Anmol

**Comment by anmode** [comment]
Some last UI fix:
1. The alignment of the message that the user sends should be on the right.
2. Provide some dark color or may be a shadow effect to the header and footer (where we write the message) so it will be distinguishable with the chat window.

**Comment by Uthra-karunakaran** [comment]
> Some last UI fix:
> 
> 1. The alignment of the message that the user sends should be on the right.
> 2. Provide some dark color or may be a shadow effect to the header and footer (where we write the message) so it will be distinguishable with the chat window.

@anmode  just pushed the commit , take a look

File: styles/form.module.css [hunk 1]
```diff
@@ -101,13 +111,20 @@
...
-  margin: 90px auto 20px;
-  width: 480px;
-  padding: 12px 45px 12px 45px;
-  box-shadow: 0 15px 20px rgba(0, 0, 0, 0.2);
+  margin: 150px auto 20px;
+  width: 100%; /*Changed from 480px to 100% for responsiveness */
+  max-width: 480px; /*Added max-width to ensure it doesn't exceed the original width */
+  padding: 12px 20px; /*Adjusted padding for better fit on smaller screens */
+  /* box-shadow: 0 15px 20px rgba(0, 0, 0, 0.2); */
+  box-shadow: 0 10px 15px rgba(0, 0, 0, 0.1); /* Reduced box shadow */
```
Comment (anmode) [inline styles/form.module.css]: please remove these comment@

File: styles/form.module.css [hunk 2]
```diff
@@ -217,15 +234,15 @@
 @media (max-width: 575px) {
   .loginform {
     width: 90%;
-    padding: 30px 40px 15px 40px;
-    margin: 100px auto 0 auto;
+    padding: 12px 20px; /* Adjusted padding for smaller screens */
+    margin: 102px auto 20px;
```
Comment (anmode) [inline styles/form.module.css]: same!

File: styles/form.module.css [hunk 3]
```diff
@@ -248,8 +265,15 @@
   }
   .btnn {
     width: 100%;
+    padding: 10px 25px; /* for better visibility of the btn text in sm screens*/
```
Comment (anmode) [inline styles/form.module.css]: same

**Review by anmode — APPROVED** [review]
Cool looks good just a minor change.

File: styles/form.module.css (hunk 3 above)
Comment (Uthra-karunakaran) [reply styles/form.module.css]: sure i'll do that

**Comment by Uthra-karunakaran** [comment]
i removed the comments anmol

**Comment by anmode** [comment]
Congrats @code-par-learn on your first PR 🚀

**Pull request merged**